tests/schemas/dump02_setup.edgeql

```diff
@@ -19,8 +19,8 @@
 
 SET MODULE default;
 
-CREATE MIGRATION m12hdldnmvzj5weaevxsmizppnl2poo6nconx2hcfkklbwcghqsmaq
-ONTO m1vyvlra26tef6oe6yu37m7lfw7i3ef3n62m6om353dvnbm3mynqqa {
+CREATE MIGRATION m1t2phsw6j2rgl4ieihm6mnvoln3ssayxncjzl2kwkxmunn2f6aqha
+ONTO m1iej6dr3hk33wykqwqgg4xxo3tivpiznpb2mto7qsw2zgipsbfihq {
```

Comment:
Why did this change?

Reply:
Probably the array thing caused things to get reordered in DDL.

Reply:
Yeah, here's the diff for the base migration:

```
diff --git a/tmp/noarr.edgeql b/tmp/arr.edgeql
index 837f90d..632379a 100644
--- a/tmp/noarr.edgeql
+++ b/tmp/arr.edgeql
@@ -6,13 +6,11 @@ CREATE FUNCTION default::`💯`(NAMED ONLY `🙀`: std::int64) ->  std::int64 {
         (100 - `🙀`)
     )
 ;};
-CREATE ABSTRACT CONSTRAINT default::`🚀🍿`(max: std::int64) EXTENDING std::max_len_value;
-CREATE TYPE default::A;
 CREATE SCALAR TYPE default::你好 EXTENDING std::str;
-CREATE SCALAR TYPE default::مرحبا EXTENDING default::你好 {
-    CREATE CONSTRAINT default::`🚀🍿`(100);
-};
+CREATE SCALAR TYPE default::مرحبا EXTENDING default::你好;
 CREATE SCALAR TYPE default::`🚀🚀🚀` EXTENDING default::مرحبا;
+CREATE ABSTRACT CONSTRAINT default::`🚀🍿`(max: std::int64) EXTENDING std::max_len_value;
+CREATE TYPE default::A;
 CREATE TYPE default::Łukasz {
     CREATE LINK `Ł💯`: default::A {
         CREATE PROPERTY `🙀مرحبا🙀`: default::مرحبا {
@@ -25,6 +23,9 @@ CREATE TYPE default::Łukasz {
     };
     CREATE INDEX ON (.`Ł🤞`);
 };
+ALTER SCALAR TYPE default::مرحبا {
+    CREATE CONSTRAINT default::`🚀🍿`(100);
+};
 CREATE TYPE default::Tree {
     CREATE REQUIRED PROPERTY val: std::str {
         CREATE CONSTRAINT std::exclusive;
@@ -47,3 +48,4 @@ CREATE TYPE default::`S p a M` {
 ALTER TYPE default::A {
     CREATE REQUIRED LINK `s p A m 🤞`: default::`S p a M`;
 };
```

Not sure what can be done here (if anything).